### PR TITLE
Adding driver for Razorbill RP100 strain cell controller

### PR DESCRIPTION
**Pull request by pheowl**
Hey, I added a driver for another instrument (probably not a very common one).

### CODE REVIEW

File: pymeasure/instruments/razorbill/__init__.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+#
+
+from .razorbillRP100 import razorbillRP100
```
Comment (bilderbuchi) [inline pymeasure/instruments/razorbill/__init__.py]: Please also add an entry about this new manufacturer to `pymeasure/instruments/__init__.py`

Also, could you please add some documentation as outlined [here](https://pymeasure.readthedocs.io/en/latest/dev/adding_instruments.html#adding-documentation)?

**Comment by pheowl** [comment]
As far as I can tell it should work now as advertised

**Comment by pheowl** [comment]
Hey, I just wanted to check whether there still is something to do here?

**Comment by bilderbuchi** [comment]
There is a merge conflict indicated that needs to be resolved. Do you know how to do that?

**Comment by bilderbuchi** [comment]
Resolved the merge conflict. Thank you for your contribution!

**Pull request merged**